pkg/codegen/schema/schema.go

```diff
@@ -319,6 +330,13 @@ type TypeSpec struct {
 	Language map[string]json.RawMessage `json:"language,omitempty"`
 }
 
+// ConstSpec is the serializable form of extra information about the constant value for a property.
+type ConstSpec struct {
+	// TODO: add Language if needed
```

Comment:
@pgavlin I'm not sure if `Language` is necessary?

Reply:
I don't think it is.